app/models/api_docs/service.rb

```diff
@@ -129,7 +129,7 @@ def prepare_base_path_notify
   end
 
   def new_base_path?
-    self.class.where({:base_path => base_path}).limit(1).empty?
+    self.class.unscoped.where({service_id: self.service_id, base_path: base_path}).limit(1).empty?
```

Comment:
It originally scopes by `service_id` + `name`, but I don't think the name is required, so in this version it's scoping only by `service_id`.

Reply:
Do we need to `limit` even though the final check is `empty?`?

Reply:
No need, but they probably did it that way so the query is faster. I thinks it's OK to leave it.